index.html

```diff
@@ -2428,58 +2428,71 @@
         </ol>
       </section>
     </section>
-    <section class='informative'>
-      <h2>
-        Security Considerations
-      </h2>
-      <p class="ednote">
-        This section is a placeholder to record security considerations as we
-        gather them through working group discussion.
-      </p>
-      <section>
+    <section class="informative">
+      <h2>Special Considerations</h2>
+      <section class='informative'>
         <h2>
-          Encryption of data fields
+          Security Considerations
         </h2>
-        <p>
-          The <a>PaymentRequest</a> API does not directly support encryption of
-          data fields. Individual <a>payment methods</a> may choose to include
-          support for encrypted data but it is not mandatory that all
-          <a>payment methods</a> support this.
+        <p class="ednote">
```

Comment:
I am not sure why this showed up in this PR, but this is not content related to the change request and I don't have a preference.  It isn't my text.